multiversion/src/main/scala/multiversion/commands/LintCommand.scala

```diff
@@ -125,6 +107,20 @@ case class LintCommand(
         }
     }
   }
+
+  private def isPending(app: Application, label: String): Boolean = {
+    BazelUtil
+      .packageRoot(app, label)
+      .map { path =>
+        val pendingFile = path.resolve("PENDING")
```

Comment:
Maybe we should also support `PENDING.bazel` to clarify that it's a Bazel build failure.